diff --git a/node_modules/react-native-splash-screen/android/src/main/java/org/devio/rn/splashscreen/SplashScreen.java b/node_modules/react-native-splash-screen/android/src/main/java/org/devio/rn/splashscreen/SplashScreen.java
index f316efe..7680d55 100644
--- a/node_modules/react-native-splash-screen/android/src/main/java/org/devio/rn/splashscreen/SplashScreen.java
+++ b/node_modules/react-native-splash-screen/android/src/main/java/org/devio/rn/splashscreen/SplashScreen.java
@@ -56,7 +56,7 @@ public class SplashScreen {
      * 打开启动屏
      */
     public static void show(final Activity activity) {
-        show(activity, false);
+        show(activity, true);
     }
 
     /**
diff --git a/node_modules/react-native-splash-screen/ios/RNSplashScreen.m b/node_modules/react-native-splash-screen/ios/RNSplashScreen.m
index 02bf3e5..c37cacf 100644
--- a/node_modules/react-native-splash-screen/ios/RNSplashScreen.m
+++ b/node_modules/react-native-splash-screen/ios/RNSplashScreen.m
@@ -25,11 +25,8 @@ + (void)show {
         [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(jsLoadError:) name:RCTJavaScriptDidFailToLoadNotification object:nil];
         addedJsLoadErrorObserver = true;
     }
-
-    while (waiting) {
-        NSDate* later = [NSDate dateWithTimeIntervalSinceNow:0.1];
-        [[NSRunLoop mainRunLoop] runUntilDate:later];
-    }
+    NSDate* later = [NSDate dateWithTimeIntervalSinceNow:3]; // 3 seconds timer
+    [[NSRunLoop mainRunLoop] runUntilDate:later];
 }
 
 + (void)showSplash:(NSString*)splashScreen inRootView:(UIView*)rootView {
